src/odemis/acq/stream/_projection.py

```diff
@@ -934,13 +945,11 @@ def _on_max_projection(self, value):
     def getBoundingBox(self):
         '''
         Get the bounding box of the whole image, whether it`s tiled or not.
+        Since the image can now be updated, the bounding box can change
+        so we need to get it from the stream directly.
         return (tuple of floats(minx, miny, maxx, maxy)): Tuple with the bounding box
         '''
-        if hasattr(self, 'rect'):
-            rng = self.rect.range
-            return rng[0][0], rng[0][1], rng[1][0], rng[1][1]
-        else:
-            return self.stream.getBoundingBox(self.image.value)
+        return self.stream.getBoundingBox()
```

Comment:
BTW, for now getBoundingBox doesn't take care of the rotation. However, for a *whole* image, that usually doesn't have huge effect, as rotation is centered on the image center. That means in the worst case, at 45° rotation, the bounding box is a little too small (but correctly centered).